src/pages/search/ListingCard.tsx

```diff
@@ -1,12 +1,17 @@
 import * as React from 'react';
 import { Card, CardBody, CardFooter, CardImg, CardSubtitle, CardText, CardTitle, Fade } from 'reactstrap';
 
+// TODO: Add a .d.ts for this, or PR one to definitelytyped?
+// @ts-ignore
+import StarRatings from 'react-star-ratings';
```

Comment:
`react-star-ratings` doesn't have types, which means [`noImplicitAny` will raise an error here](https://github.com/Microsoft/TypeScript/issues/13348#issuecomment-405361558) unless we [add and maintain a `.d.ts`](http://www.typescriptlang.org/docs/handbook/modules.html#shorthand-ambient-modules) with declarations for it.

Doing a `@ts-ignore` seems preferable to me, given the usage here, but I'm glad to go the `.d.ts` route (either here or in a subsequent PR) if anyone would prefer that

Reply:
is it possible to declare a new module in the d.ts and make it `content: any`? in case we use the starts elsewhere (maybe listing page?)

Reply:
Adding custom typings (and a `typings` directory for any future cases we need to do this). It was a bit tricky getting all parts of the configuration right, but at least it ended up being a small(-ish) change set.

Went ahead and defined the Props interface for `react-star-ratings`; found it to be a useful exercise, and if we're going to expend effort satisfying TypeScript, might as well get the benefits!